packages/server/src/app.ts

```diff
@@ -61,7 +61,6 @@ if (env.isProd()) {
 
 const server = http.createServer(app.callback())
 destroyable(server)
-initialiseWebsockets(app, server)
```

Comment:
Websocket initialisation has been moved to `startup.ts`, as it needs to be done after redis has been initialised, so that we can use the socket.io redis adapter.